app-backend/api/src/main/scala/organization/Routes.scala

```diff
@@ -35,15 +35,15 @@ trait OrganizationRoutes extends Authentication with PaginationDirectives with U
     }
   }
 
-  def listOrganizations: Route = authenticate { user =>
+  def listOrganizations: Route = authenticateRootMember { root =>
```

Comment:
I've posted an attempt at this in https://github.com/azavea/raster-foundry/pull/1383/commits/4ba70b3c8f85b11fa6b0d3b7fdae66bea835ebd7. Does this look good?